server/chainquery/models/AbnormalClaimModel.js

```diff
@@ -0,0 +1,65 @@
+const getterMethods = {
+  // Add as needed, prefix all methods with `generated`
+}
+
+export default (sequelize, {
+  STRING, BOOLEAN, INTEGER, TEXT, DECIMAL
+}) => sequelize.define(
+  'abnormal_claim',
+  {
+    id: {
+      primaryKey: true,
+      type: INTEGER,
+      set() { },
+    },
+    name: {
+      type: STRING,
+      set() { },
+    },
+    claim_id: {
+      type: STRING,
+      set() { },
+    },
+    is_update: {
+      type: INTEGER,
+      set() { },
+    },
+    block_hash: {
+      type: STRING,
+      set() { },
+    },
+    transaction_hash: {
+      type: STRING,
+      set() { },
+    },
+    vout: {
+      type: INTEGER,
+      set() { },
+    },
+    output_id: {
+      type: INTEGER,
+      set() { },
+    },
+    value_as_hex: {
+      type: TEXT,
+      set() { },
+    },
+    value_as_json: {
+      type: TEXT,
+      set() { },
+    },
+    created_at: {
+      type: INTEGER,
```

Comment:
These are timestamps, is it ok to create it as an integer?

Reply:
This will work for now, I need to verify that `Sequelize.DATE(6)` is fully compatible first.
There's a few things like `INT(1)` that I'm using `INTEGER` for as well that could probably be switched to `Sequelize.BOOLEAN`.  I know there's a number of `BIGINT` as well that I need to adjust.